Cargo.toml

```diff
@@ -17,226 +17,244 @@ members = [
+polkadot-runtime-metrics = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+polkadot-runtime-parachains = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+polkadot-service = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+polkadot-statement-distribution = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+polkadot-statement-table = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+slot-range-helper = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+tracing-gum = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+tracing-gum-proc-macro = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+xcm = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+xcm-builder = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+xcm-executor = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+xcm-procedural = { git = "https://github.com/paritytech//polkadot", branch = "locked-for-gav-xcm-v3-and-bridges-3" }
+
+[patch."https://github.com/paritytech/cumulus"]
+cumulus-client-consensus-aura = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-client-consensus-common = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-client-cli = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-client-network = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-client-service = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-primitives-core = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-primitives-parachain-inherent = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-relay-chain-interface = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-relay-chain-inprocess-interface = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-pallet-aura-ext = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-pallet-parachain-system = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-pallet-dmp-queue = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-pallet-xcm = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
+cumulus-pallet-xcmp-queue = { git = "https://github.com/paritytech//cumulus", branch = "gav-xcm-v3" }
```

Comment:
`rialto-parachain` uses `cumulus/master` and rest use `cumulus/gav-xcm-v3`, why is that?

Reply:
everywhere should be just master, but this is the way that to tell cargo to use `branch = "gav-xcm-v3" instead of `master`,

at the end, everywhere should be master, and this patch part will be removed